js/flipcounter.js

```diff
@@ -165,6 +176,8 @@ var flipCounter = function(d, options){
     _digitCheck(x, y);
     // Do first animation
     if (counter.auto === true) nextCount = setTimeout(_doCount, counter.pace);
+    // Stop counter
+    if (counter.value === counter.stop) _clearNext();
```

Comment:
`===` forces you to have an exact value to stop the counter.
I suggest `if (counter.value >= counter.stop) _clearNext();` to stop the counter.